src/web/directories.cc

```diff
@@ -87,16 +89,21 @@ void Web::Directories::process()
 
         /// \todo replace hexEncode with base64_encode?
         std::string id = hexEncode(filepath.c_str(), filepath.string().length());
-        filesMap.emplace(id, std::pair(filepath.filename(), hasContent));
+        filesMap.emplace(id, std::pair(filepath, hasContent));
     }
 
     auto f2i = StringConverter::f2i(config);
     for (auto&& [key, val] : filesMap) {
-        auto&& [file, has] = val;
+        auto file = val.first;
```

Comment:
:) Curious about this. I was under the impression structured bindings we're passed by value.

Reply:
I tried several options in vain.